recipes/grpcio/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "grpcio" %}
+{% set version = "1.6.3" %}
+{% set sha256 = "e9662782c58bc21be26163b78136eaed091dfd45fac699cb711ee9eeeb7e2f9b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+  run:
+    - python
+    - six >=1.5.2
+    - protobuf >=3.3.0
+    - futures >=2.2.0
```

Comment:
It seems like `futures` and `enum34` are only required for python2, so you should add the appropriate tags. See example here:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors